lcm/dircon_saved_trajectory.cc

```diff
@@ -78,6 +78,24 @@ DirconTrajectory::DirconTrajectory(
       lambda_c_.push_back(&collocation_force_traj);
     }
 
+    // Collocation slack vars
+    if (state_breaks[mode].size() > 1) {
+      LcmTrajectory::Trajectory collocation_slack_traj;
+      collocation_slack_traj.traj_name =
+          "collocation_slack_vars" + std::to_string(mode);
+      collocation_slack_traj.datatypes = collocation_force_names;
+      collocation_slack_traj.time_vector =
```

Comment:
We sure we want to change this dairlib function instead of writing our own lib?

Reply:
Yeah. I'm just filling in the functionality of the dairlib class.